nx.json

```diff
@@ -3,7 +3,6 @@
   "nxCloudAccessToken": "YjIzMmMxMWItMjhiMS00NWY2LTk1NWYtYWU3YWQ0YjE4YjBlfHJlYWQ=",
   "release": {
     "projects": [
-      "*",
```

Comment:
This is no longer needed, we can use a list of only negative matchers and the ALL EXCEPT is implied